backend/views/syson-diagram-common-view/src/main/java/org/eclipse/syson/diagram/common/view/services/ViewNodeService.java

```diff
@@ -145,4 +153,34 @@ public boolean isHiddenByDefault(Element self, String referenceName) {
         }
         return isHiddenByDefault;
     }
+
+    /**
+     * Check if a Start Action can be added to the given element.
+     *
+     * @param self
+     *            a {@link Namespace} element
+     * @return <code>true</code> if the Start action can be added to the given element or <code>false</code> otherwise.
+     */
+    public boolean canAddStartAction(Namespace self) {
+        // 1. Check that self is an Action usage or definition
+        if (self instanceof ActionUsage || self instanceof ActionDefinition) {
```

Comment:
this test seems useless because `getStandardStartAction` does the exact same test on first line.

Reply:
Correct, I've changed.